core/templates/cowdata.h

```diff
@@ -72,21 +72,11 @@ class CowData {
 private:
 	// Function to find the next power of 2 to an integer.
 	static _FORCE_INLINE_ USize next_po2(USize x) {
-		if (x == 0) {
-			return 0;
-		}
-
-		--x;
-		x |= x >> 1;
-		x |= x >> 2;
-		x |= x >> 4;
-		x |= x >> 8;
-		x |= x >> 16;
 		if (sizeof(USize) == 8) {
```

Comment:
It won't make a difference probably, but `if constexpr` is definitely warranted while you're at it